src/serializers/serializeStaticFilters.ts

```diff
@@ -1,32 +1,41 @@
 import { isNumberRangeValue } from '../models/searchservice/common/NumberRangeValue';
-import { CombinedFilter } from '../models/searchservice/request/CombinedFilter';
-import { Filter } from '../models/searchservice/request/Filter';
+import { FilterCombinator, StaticFilter } from '../models/searchservice/request/StaticFilter';
+import { FieldValueFilter } from '../models/searchservice/request/FieldValueFilter';
 import { ApiStaticFilters, ApiFilter } from '../models/searchservice/request/ApiStaticFilters';
 
 export function serializeStaticFilters(
-  filter: CombinedFilter | Filter): string | undefined {
-  if (isCombinedFilter(filter)) {
-    return JSON.stringify(shapeCombinedFilterForApi(filter));
+  filter: StaticFilter): string | undefined {
+  if (filter.kind === 'combination') {
+    const shapedFilter = shapeCombinedFilterForApi(filter.combinator, filter.children);
+    return shapedFilter && JSON.stringify(shapedFilter);
   }
 
-  return JSON.stringify(shapeFilterForApi(filter));
+  return JSON.stringify(shapeFieldValueFilterForApi(filter.value));
 }
 
-function shapeCombinedFilterForApi(combinedFilter: CombinedFilter): ApiStaticFilters {
+function shapeCombinedFilterForApi(
+  combinator: FilterCombinator,
+  filters: StaticFilter[]
+): ApiStaticFilters | undefined {
```

Comment:
Can the method declaration fit on one line?

Reply:
no, eslint complains about the line being too long if it's all on the same one